### DISCUSSION TITLE
Suggestion for some UI interaction

### DISCUSSION THREAD

**Discussion by kitepad** [stefankueng/BowPad#210] [General]
Currently, following functions is accessed by ribbon or command palette, I suggest it can access from status bar.
1. Encoding change. (convert to, load as) , launch encoding dialog when click in part of status bar or clicked in command palette.
2. Code Style change, launch lexers list dialog when clicked in doc_type part of status bar.
3. Goto line. (show line number input control when click line part in status bar)

**Comment by stefankueng** [comment]
created an issue for this: #211

**Comment by stefankueng** [comment]
1. this is not really feasible, since both convert to and load as consist of huge menus. Showing those from the status bar would require implementing this twice separately
2. same here - also these menus are not dialogs that can be just shown outside the ribbon
3. this however is possible
